cookiecutter-django-app/hooks/post_gen_project.py

```diff
@@ -12,11 +12,7 @@
 # cookiecutter can import a template from either github or from a location on local disk.
 # If someone is debugging this repository locally, the below block is necessary to pull in
 #   local versions of the templates
-EDX_COOKIECUTTER_ROOTDIR = os.getenv('EDX_COOKIECUTTER_ROOTDIR')
-import_template_from_github = True
-if EDX_COOKIECUTTER_ROOTDIR is not None and isinstance(EDX_COOKIECUTTER_ROOTDIR, str):
-    if len(EDX_COOKIECUTTER_ROOTDIR) > 0:
-        import_template_from_github = False
+EDX_COOKIECUTTER_ROOTDIR = os.getenv('EDX_COOKIECUTTER_ROOTDIR') or 'https://github.com/edx/edx-cookiecutters.git'
```

Comment:
With the refactoring below, we should be able to just treat this as a generic path; it can be local or remote via github and cookiecutter will handle it just the same :)
So with this one line, if the variable is set (and not empty!) cookiecutter will load sub-templates from there, but otherwise will default to github.

In theory, this change would technically _also_ let you set the variable to _another_ repo, so you could load sub-templates from a different repo.
I don't know if or why you'd want to do that, but you could ;)